data/campaigns/Secrets_of_the_Ancients/scenarios/14_Entering_the_Northlands.cfg

```diff
@@ -28,7 +28,7 @@
             title_alignment=left
             story= _ "24 IV, 23 YW
 
-We reached the Ford of Abez as the dawn light spread behind the mountain peaks to the north. As we waded across, the dark water reached to our waists, which made for slow going. I dared not imagine what manner of tentacled creature lurked below, but we completed the crossing without incident. The water was chill with snowmelt, so our teeth were chattering when we reached the far shore. The rising sun was warm though, and had us dried in short order. Despite this, Ras-Tabahn was not well off. He collapsed in the sand and went into a paroxysm of coughing the like of which I had never seen. When he could speak again, he assured me that it was only on account of the cold, and I should not be unduly troubled. I am uneasy all the same."
+We reached the Ford of Abez as the dawn light spread behind the mountain peaks to the north. As we waded across, the dark water reached up to our waists, which made for slow going. I dared not imagine what manner of tentacled creature lurked below, but we completed the crossing without incident. The water was chill with snowmelt, so our teeth were chattering when we reached the far shore. The rising sun was warm though, and had us dried in short order. Despite this, Ras-Tabahn was not well. He collapsed in the sand and went into a paroxysm of coughing the like of which I had never seen. When he could speak again, he assured me that it was only on account of the cold, and I should not be unduly troubled. I am uneasy all the same."
```

Comment:
This line isn't necessary to change, I think the original is fine.

Reply:
Often I find 'up' can be redundant but perhaps not in this case. But it's okay for me without it.

The other change I think is a good one. If you don't like it, how about 'Ras-Tabahn was not faring well'?

Reply:
I'd rather have Ardonna talk plainly. She seems like she wouldn't use extra words when it's not necessary.

Reply:
In that case, all the more reason to just say 'Ras-Tabahn was not well'.

Reply:
Will accept removing the 'up', but can we remove the 'off'? My understanding is that anytime you say 'Well off' it means 'rich'.

Reply:
Oops, I was super tired while reading this apparently. Yeah, please remove the off.

Reply:
That is already done, so marking as resolved. I don't think there are any outstanding changes pending.